README.md

```diff
@@ -7,8 +7,8 @@
 - Manage storage & databases with a unified Python API ("lakehouse").
 - Track data lineage across notebooks & pipelines.
 - Integrate registries for experimental metadata & in-house ontologies.
-- Validate, standardize & annotate.
-- Collaborate across a mesh of distributed LaminDB instances.
+- Validate, standardize & annotate metadata.
```

Comment:
We don't annotate metadata, we annotate data.

I'd not add "metadata", here. We had "data" here before it felt quite repetitive (what else would you validate, standardize & annotate).

Reply:
Whops, good catch. Yes of course